spyder/widgets/tests/test_sidebardialog.py

```diff
@@ -55,10 +56,15 @@ def setup_page(self):
     class TestDialog(SidebarDialog):
         PAGE_CLASSES = [Page1, Page2]
 
-    qapp.setStyleSheet(str(APP_STYLESHEET))
+    if running_in_ci():
```

Comment:
```suggestion
    if not running_in_ci():
```

This was my mistake, sorry.